samples/MvvmSampleUwp/MvvmSampleUwp/Assets/docs/Ioc.md

```diff
@@ -1,47 +1,82 @@
 ---
 title: Ioc
 author: Sergio0694
-description: A type that facilitates the use of the IServiceProvider type
+description: An introduction to the use of the IServiceProvider type through the Microsoft.Extensions.DependencyInjection APIs
 keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, mvvm, service, dependency injection, net core, net standard
 dev_langs:
   - csharp
 ---
 
 # Ioc ([Inversion of control](https://en.wikipedia.org/wiki/Inversion_of_control))
 
-The [Ioc](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.mvvm.DependencyInjection.Ioc) class is a type that facilitates the use of the `IServiceProvider` type. It's powered by the `Microsoft.Extensions.DependencyInjection` package, which provides a fully featured and powerful DI set of APIs, and acts as an easy to setup and use `IServiceProvider`.
+A common pattern that can be used to increase modularity in the codebase of an application using the MVVM pattern is to use some form of inversion of control. One of the most common solution in particular is to use dependency injection, which consists in creating a number of services that are injected into backend classes (ie. passed as parameters to the viewmodel constructors) - this allows code using these services not to rely on implementation details of these services, and it also makes it easy to swap the concrete implementations of these services. This pattern also makes it easy to make platform-specific features available to backend code, by abstracting them through a service which is then injected where needed.
+
+The MVVM Toolkit doesn't provide built-in APIs to facilitate the usage of this pattern, as there already exist dedicated libraries specifically for this such as the `Microsoft.Extensions.DependencyInjection` package, which provides a fully featured and powerful DI set of APIs, and acts as an easy to setup and use `IServiceProvider`. The following guide will refer to this library and provide a series of examples of how to integrate it into applications using the MVVM pattern.
```

Comment:
```suggestion
The MVVM Toolkit doesn't provide built-in APIs to facilitate the usage of this pattern, as dedicated libraries specifically exist for this, such as the `Microsoft.Extensions.DependencyInjection` package. It provides a fully featured and powerful set of dependency injection APIs already and can be easily setup to use the `IServiceProvider` interface. The following guide will refer to this library and provide a series of examples of how to integrate it into applications using the MVVM pattern with the MVVM Toolkit.
```

Reply:
Can we also provide the doc link to the IServiceProvider?

Reply:
Yup, done in 8b39c671b0e3adb049c6ca2f4d0ee88bc1a54277, thanks! 😄